content/projects/blueocean/roadmap/index.html.haml

```diff
@@ -0,0 +1,99 @@
+---
+layout: default
+title: Blue Ocean Roadmap
+css:
+  - '/assets/bower/bootstrap/css/bootstrap.min.css'
+  - '/css/blueocean-animate.css'
+  - '/css/blueocean-roadmap-style.css'
+---
+%header#header.header
+  .skew
+    .container.wow.fadeIn{"data-wow-delay" => "0.5s"}
+      .row
+        .header-inner.col-lg-12
+          .logo
+            %svg.logo-jenkins{:height => "102px", :version => "1.1", :viewbox => "682 209 74 102", :width => "74px", :xmlns => "http://www.w3.org/2000/svg", "xmlns:xlink" => "http://www.w3.org/1999/xlink"}
```

Comment:
I'll change this, but for future reference please pull in the logo partial instead of copy/pasting the SVG haml all over the joint